src/storage/v2/mvcc.hpp

```diff
@@ -113,18 +116,35 @@ inline Delta *CreateDeleteObjectDelta(Transaction *transaction) {
                                            transaction->command_id);
 }
 
-/// TODO: what if in-memory analytical
-inline Delta *CreateDeleteDeserializedObjectDelta(Transaction *transaction, std::optional<std::string> old_disk_key) {
+inline Delta *CreateDeleteObjectDelta(Transaction *transaction, std::list<Delta> *deltas) {
```

Comment:
Should we check inside for `transaction->storage_mode == StorageMode::IN_MEMORY_ANALYTICAL` as it's done for `CreateDeleteObjectDelta(Transaction *transaction)` ?

Reply:
Not necessary but will add the check